codegen/validation.go

```diff
@@ -477,7 +485,14 @@ if {{ if .string }}utf8.RuneCountInString({{ $target }}){{ else }}len({{ $target
 }
 {{- end }}`
 
-	requiredValTmpl = `if {{ $.target }}.{{ .attCtx.Scope.Field $.reqAtt .req true }} == nil {
-        err = goa.MergeErrors(err, goa.MissingFieldError("{{ .req }}", {{ printf "%q" $.context }}))
-}`
+	requiredValTmpl = `if {{ $.target }}.{{ .attCtx.Scope.Field $.reqAtt .req  true }} == nil {
```

Comment:
@nitinmohan87 @raphael I hope you don't mind me tagging you, but I'm a little stumped on how to handle the validation templates. This is sort of a first try at it, going off of the validation set out in the issue about this, it's currently not working :) I haven't really messed about with Go templates before (had a look at the documentation, but that's all my experience with it) so it would be great if you could give me some pointers :D